# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/*/isdn4k-utils/multi-line-strings.patch
# Copyright (C) 1998 - 2003 ROCK Linux Project
# 
# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 2 of the License, or
# (at your option) any later version. A copy of the GNU General Public
# License can be found at Documentation/COPYING.
# 
# Many people helped and are helping developing ROCK Linux. Please
# have a look at http://www.rocklinux.org/ and the Documentation/TEAM
# file for details.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

diff -urN -x '*~' isdn4k-utils/ipppd/options.c isdn4k-utils-gcc-3.3.1/ipppd/options.c
--- isdn4k-utils-old/ipppd/options.c	2002-07-06 02:34:08.000000000 +0200
+++ isdn4k-utils-new/ipppd/options.c	2003-08-19 22:36:18.000000000 +0200
@@ -479,26 +479,25 @@
 #define IMPLEMENTATION ""
 #endif
 
-static char *usage_string = "\
-ipppd version %s patch level %d%s\n\
-Usage: %s [ options ], where options are:\n\
-\t<device>	Communicate over the named device\n\
+static char *usage_string = ""
+"ipppd version %s patch level %d%s\n"
+"Usage: %s [ options ], where options are:\n"
+"\t<device>	Communicate over the named device\n"
 #ifdef INCLUDE_OBSOLETE_FEATURES
-\tcrtscts		Use hardware RTS/CTS flow control\n\
-\t<speed>		Set the baud rate to <speed>\n\
-\tmodem		Use modem control lines\n\
+"\tcrtscts		Use hardware RTS/CTS flow control\n"
+"\t<speed>		Set the baud rate to <speed>\n"
+"\tmodem		Use modem control lines\n"
 #endif
-\t<loc>:<rem>	Set the local and/or remote interface IP\n\
-\t\taddresses.  (you also may use the option 'useifip' to get IPs).\n\
-\tasyncmap <n>	Set the desired async map to hex <n>\n\
-\tauth		Require authentication from peer\n\
-\tconnect <p>     Invoke shell command <p> to set up the serial line\n\
-\tdefaultroute	Add default route through interface\n\
-\tfile <f>	Take options from file <f>\n\
-\tmru <n>		Set MRU value to <n> for negotiation\n\
-\tnetmask <n>	Set interface netmask to <n>\n\
-See ipppd(8) for more options.\n\
-";
+"\t<loc>:<rem>	Set the local and/or remote interface IP\n"
+"\t\taddresses.  (you also may use the option 'useifip' to get IPs).\n"
+"\tasyncmap <n>	Set the desired async map to hex <n>\n"
+"\tauth		Require authentication from peer\n"
+"\tconnect <p>     Invoke shell command <p> to set up the serial line\n"
+"\tdefaultroute	Add default route through interface\n"
+"\tfile <f>	Take options from file <f>\n"
+"\tmru <n>		Set MRU value to <n> for negotiation\n"
+"\tnetmask <n>	Set interface netmask to <n>\n"
+"See ipppd(8) for more options.\n";
 
 static char *current_option;   /* the name of the option being parsed */
 
